ciao-controller/openstack_compute.go

```diff
@@ -373,6 +373,7 @@ func (c *controller) CreateServer(tenant string, server compute.CreateServerRequ
 	if err != nil {
 		return server, err
 	}
+	volumes := abstractBlockDevices(blockDeviceMappings)
```

Comment:
I notice that this would make this particular commit not compile if we had to bisect. maybe rearrange if possible to improve bisectability.